pkg/crc/machine/stop.go

```diff
@@ -25,3 +31,26 @@ func (client *client) Stop() (state.State, error) {
 	}
 	return host.Driver.GetState()
 }
+
+// This should be removed after https://bugzilla.redhat.com/show_bug.cgi?id=1965992
+// is fixed. We should also ignore the openshift specific errors because stop
+// operation shouldn't depend on the openshift side.
```

Comment:
I'd mention somewhere that without this workaround, shutting down the VM takes more than 8 minutes.